src/domains/sign/io_sign_backoffice_func.tf

```diff
@@ -139,3 +139,24 @@ resource "azurerm_monitor_autoscale_setting" "io_sign_backoffice_func" {
     }
   }
 }
+
+resource "azurerm_private_endpoint" "io_sign_backoffice_func" {
+  name                = format("%s-backoffice-endpoint", local.project)
+  location            = azurerm_resource_group.data_rg.location
+  resource_group_name = azurerm_resource_group.data_rg.name
+  subnet_id           = data.azurerm_subnet.private_endpoints_subnet.id
+
+  private_service_connection {
+    name                           = format("%s-backoffice-endpoint", local.project)
```

Comment:
make sure that this `name` is unique (isn't `%s-backoffice-endpoint` already used for the backoffice `app service`?)